tests/models/test_models.py

```diff
@@ -12,33 +10,19 @@ class TestALNModel(unittest.TestCase):
     """
 
     def test_single_node(self):
-        logging.info("\t > ALN: Testing single node ...")
-        start = time.time()
-
```

Comment:
No output and measurements for testing anymore?

Reply:
when testing with a suite (`unittest` or `pytest`), you don't need to: you can't see the output anyway and testing is timing everything by default, you can later check the results so no need to do it explicitely within the testing script

Reply:
so e.g. `pytest --durations=0 tests/` shows:
```
=========================================================================================== test session starts ============================================================================================
platform darwin -- Python 3.7.6, pytest-5.3.4, py-1.8.1, pluggy-0.13.1
rootdir: /Users/nikola/work-brain/neurolib
collected 19 items

tests/models/test_models.py ....                                                                                                                                                                     [ 21%]
tests/utils/test_connectivity.py ...............                                                                                                                                                     [100%]

============================================================================================= warnings summary =============================================================================================
tests/models/test_models.py::TestALNModel::test_network
tests/models/test_models.py::TestALNModel::test_network
....
-- Docs: https://docs.pytest.org/en/latest/warnings.html
========================================================================================== slowest test durations ==========================================================================================
15.41s call     tests/models/test_models.py::TestALNModel::test_network
6.02s call     tests/models/test_models.py::TestHopfModel::test_network
3.09s call     tests/models/test_models.py::TestALNModel::test_single_node
0.68s call     tests/models/test_models.py::TestHopfModel::test_single_node
0.03s call     tests/utils/test_connectivity.py::TestFunctionalConnectivity::test_init_from_timeseries
0.01s call     tests/utils/test_connectivity.py::TestBaseConnectivity::test_load_from_file

(0.00 durations hidden.  Use -vv to show these durations.)
```

Reply:
Perfect!